met-api/migrations/versions/c09e77fde608_added_enum_value_for_timeline_widget.py

```diff
@@ -0,0 +1,61 @@
+"""Added enum value for Timeline Widget.
+
+Revision ID: c09e77fde608
+Revises: 3e4dc76a96ab
+Create Date: 2023-12-06 11:46:20.934373
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'c09e77fde608'
+down_revision = '3e4dc76a96ab'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('timeline_event', sa.Column('created_date', sa.DateTime(), nullable=False))
+    op.add_column('timeline_event', sa.Column('updated_date', sa.DateTime(), nullable=True))
+    op.add_column('timeline_event', sa.Column('created_by', sa.String(length=50), nullable=True))
+    op.add_column('timeline_event', sa.Column('updated_by', sa.String(length=50), nullable=True))
+    
+    op.add_column('widget_timeline', sa.Column('created_date', sa.DateTime(), nullable=False))
+    op.add_column('widget_timeline', sa.Column('updated_date', sa.DateTime(), nullable=True))
+    op.add_column('widget_timeline', sa.Column('created_by', sa.String(length=50), nullable=True))
+    op.add_column('widget_timeline', sa.Column('updated_by', sa.String(length=50), nullable=True))
+
+    widget_type_table = sa.table('widget_type',
+        sa.Column('id', sa.Integer),
+        sa.Column('name', sa.String),
+        sa.Column('description', sa.String))
+
+    op.bulk_insert(
+        widget_type_table,
+        [
+            {
+                'id': 8,
+                'name': 'CAC Form',
+                'description': 'Add a CAC Form to your project',
```

Comment:
I imagine this example text should be changed? CAC forms are unrelated to timelines, right?

Reply:
So, in met-web, for the widget type ENUM, value 8 is reserved for CAC forms. Timeline is 9. When I looked in the DB, CAC Form was missing as a widget type, so I added it as number 8 and added Timeline as number 9. I thought it would be better to leave that path open, rather than remove all instances of CAC Form.